### PR TITLE
css responsive

### PR DESCRIPTION
**Pull request by Valcis**

### CODE REVIEW

**Comment by doboapps** [comment]
All right! 👍

File: staff/pedrovalverde/css-layout-adaptive/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+@import url("mobile.css") (max-width: 768px);
+@import url("desktop.css") (min-width: 768px);
```
Comment (agandia9) [inline staff/pedrovalverde/css-layout-adaptive/styles/main.css]: there are common properties in both versions (mobile and desktop), and should be in the main, so we shortened the length of both CSS files.

File: staff/pedrovalverde/css-layout-adaptive/styles/mobile.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+body, html{
+    height: 100%;
+}
```
Comment (agandia9) [inline staff/pedrovalverde/css-layout-adaptive/styles/mobile.css]: background-color: black; is repeated in all the selectors of your file, like other properties ... should you group properties?

Comment (Valcis) [reply staff/pedrovalverde/css-layout-adaptive/styles/mobile.css]: yes, I should. thanks 👍

File: staff/pedrovalverde/css-layout-adaptive/styles/main.css (hunk 1 above)
Comment (Valcis) [reply staff/pedrovalverde/css-layout-adaptive/styles/main.css]: done!

**Comment by manuelbarzi** [comment]
👍

**Pull request merged**